.github/actions/setup-rust/action.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright © SixtyFPS GmbH <info@slint-ui.com>
+# SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+# cSpell: ignore debuginfo rustflags swatinem
+
+---
+name: Setup Rust support
+description: Set up Slint development environment
+
+inputs:
+  toolchain:
+    description: "Rust toolchain to use (default: stable)"
+    required: false
+    default: stable
+  target:
+    description: "Rust target to use"
+    required: false
+    default: ""
+  components:
+    description: "Rust components to install"
+    required: false
+    default: ""
+  key:
+    description: "Extra cache keying information"
+    required: false
+    default: ""
+
+runs:
+  using: composite
+  steps:
+    - name: Disable debug info to avoid running out of disk space on Windows
+      if: runner.os == 'Windows'
+      run: |
+        echo "RUSTFLAGS=-C debuginfo=0" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
```

Comment:
This means the binary we shipped won't have debug info. 
I dont't know if they had it before this, since we build with --release anyway.

Reply:
I moved this into setup_rust since it seemed to be likely that it can blow up tests. It had already blown up one test to get added in the first place:-)